utils/generate_types/test/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "target": "ES2019",
     "noEmit": true,
+    "resolveJsonModule": true,
```

Comment:
Can we read it instead of import with require? Thats how we do it for deviceDescriptors

Reply:
I've grepped the repo for `deviceDescriptors`, but only found requires/imports. Could you point me to what you are thinking of?

Reply:
(edited my message, I meant if require works instead of import)